apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.html

```diff
@@ -12,26 +12,29 @@
         </fd-input-group>
     </div>
     <div class="fd-docs-sidebar-content">
-        <nav *ngFor="let section of sections">
-            <ul class="nav">
-                <div class="side-nav__headers">{{ section.header }}</div>
-                <ng-container *ngFor="let item of section.content">
-                    <li *ngIf="item.url === 'home' || (item | docsSearch: search)">
-                        <div class="nav-item-container">
-                            <a
-                                class="nav-item"
-                                [routerLink]="'/' + item.url"
-                                (keypress)="onKeypressHandler($event)"
-                                tabindex="0"
-                                role="button"
-                                routerLinkActive="nav-item--active"
-                            >
-                                {{ item.name }}
-                            </a>
-                        </div>
-                    </li>
-                </ng-container>
-            </ul>
-        </nav>
+        <fd-side-nav>
+            <div fd-side-nav-main>
+                <ul fd-nested-list [textOnly]="true">
```

Comment:
Should't this be in reverse order?
`nav` -> `ul` -> `li`
I think you need to consult @JKMarkowski, he is preparing an update for side-nav and some markup might change.